build.sbt

```diff
@@ -83,3 +67,56 @@ lazy val example = project
       "org.http4s" %%% "http4s-circe" % http4sVersion
     )
   )
+
+lazy val testServer = project
+  .in(file("test-server"))
+  .enablePlugins(NoPublishPlugin)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.typelevel" %% "cats-effect" % catsEffectVersion,
+      "org.http4s" %% "http4s-dsl" % http4sVersion,
+      "org.http4s" %% "http4s-ember-server" % http4sVersion,
+      "ch.qos.logback" % "logback-classic" % "1.2.6",
+    )
+  )
```

Comment:
I also use a test server in http4s-dom, but I run it as part of the meta-build.

https://github.com/http4s/http4s-dom/blob/51c42d18f0d5a7d8ebc79bb47e0960bb4bfc6b35/build.sbt#L47-L48

It's not perfect but it works.

Reply:
I'm not sure if I got your point, could you please elaborate?